scio-examples/src/main/scala/com/spotify/scio/examples/extra/SortMergeBucketExample.scala

```diff
@@ -193,6 +193,11 @@ object SortMergeBucketTransformExample {
   def pipeline(cmdLineArgs: Array[String]): ScioContext = {
     val (sc, args) = ContextAndArgs(cmdLineArgs)
 
+    implicit val coderUserData: Coder[GenericRecord] =
```

Comment:
it is too bad that the SMB read/transform APIs now require Coders just to execute the `isTest` path :( But I can't see a way around that.